src/app/details/_smart-components/recipe-form/recipe-form.component.ts

```diff
@@ -53,33 +51,12 @@ export class RecipeFormComponent
     if (this.form.valid) {
       const formData = this.form.getRawValue();
       if (this.editMode && this.recipeData) {
```

Comment:
jeśli editMode jest true kiedy recipeData jest true to chyba nie potrzebne to drugie

Reply:
null safety :)